pages/src/footer/footer.less

```
@@ -32,8 +36,8 @@
     justify-content: center;
     cursor: pointer;
     margin-left: 30px;
-    width: 150px;
-    height: 35px;
+    width: 180px;
+    height: 55  px;
```

```suggestion
    height: 55px;
```
Extra spaces, that may cause an issues